src/ParaTest/Runners/PHPUnit/ResultPrinter.php

```diff
@@ -201,16 +201,6 @@ public function printFeedback(ExecutableTest $test)
                 $test->getPath()
             ));
         }
-        if (!$reader->hasResults()) {
```

Comment:
I don't think we can safely remove this condition since it's designed to detect certain edge cases (like a process crashed, or there is wrongly more than one class in the file, etc...)